src/main/webapp/video.js

```diff
@@ -45,3 +43,18 @@ function onYouTubeIframeAPIReady() {
 function onPlayerReady(event) {
   event.target.playVideo();
 }
+
+/** Starts timer if {@code currentState} is playing, stops otherwise. */
+function startStopTimerOnStateChange(currentState) {
```

Comment:
nit: I would name this method startOrStopTimer(). You can omit the state change because you can deduce that from the input.